src/pudl/output/__init__.py

```diff
@@ -12,6 +12,8 @@
 """
 from . import (  # noqa: F401
     censusdp1tract,
+    denorm_eia,
+    denorm_ferc,
```

Comment:
I think you mean `denorm_ferc1` here, since there are other FERC forms that will have their own assets probably.

Reply:
Ah ok. Updated.